packages/wrangler/templates/pages-dev-pipeline.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ *
+ */
+
+// @ts-ignore entry point will get replaced
+import worker from "__ENTRY_POINT__";
+// @ts-ignore entry point will get replaced
+export * from "__ENTRY_POINT__";
+
+const transformToRegex = (filter: string) => {
+	return filter.replace("*", ".*");
+};
+
+const routes = {
+	// @ts-ignore routes are injected
+	include: __ROUTES__.include.map(transformToRegex),
+	// @ts-ignore routes are injected
+	exclude: __ROUTES__.exclude?.map(transformToRegex) || [],
```

Comment:
`exclude` isn't optional, so why do we need the `?`?

Reply:
we don't! I think I just forgot it in there. good catch!

Reply:
✅